kimchi/src/circuits/constraints.rs

```diff
@@ -291,25 +247,6 @@ pub fn vanishes_on_last_4_rows<F: FftField>(domain: D<F>) -> DP<F> {
     &(&(&x - &c(w1)) * &(&x - &c(w2))) * &(&(&x - &c(w3)) * &(&x - &c(w4)))
 }
```

Comment:
these are functions that are only related to the permutation, so I think they should be moved to permutation.rs

Reply:
just so im following we are removing `vanishes_on_last_4_rows` to permutation.rs ya?

Reply:
and `eval_vanishes_on_last_4_rows` as well. Although it seems to be outside the scope of the first issue, I was just commenting as I noticed these :)

Reply:
no worries. just wanted to be sure not to break things :p